Framework/Algorithms/src/GetDetectorOffsets.cpp

```diff
@@ -53,8 +55,10 @@ void GetDetectorOffsets::init() {
                   "generated OffsetsWorkspace.");
   declareProperty(std::make_unique<WorkspaceProperty<OffsetsWorkspace>>("OutputWorkspace", "", Direction::Output),
                   "An output workspace containing the offsets.");
-  declareProperty(std::make_unique<WorkspaceProperty<>>("MaskWorkspace", "Mask", Direction::Output),
-                  "An output workspace containing the mask.");
+  declareProperty(std::make_unique<WorkspaceProperty<MaskWorkspace>>("MaskWorkspace", "", Direction::Output),
```

Comment:
I like that this is now an explicit output. It would be good if it could be optional.